tests/foreign_object/tests.py

```diff
@@ -412,3 +412,28 @@ def test_foreign_object_form(self):
         a2 = form.save()
         self.assertEqual(a.pk, a2.pk)
         self.assertEqual(a2.pub_date, datetime.date(2013, 1, 1))
+
+
+class RestrictedConditionsTests(TestCase):
+
+    def setUp(self):
+        for i in range(10):
+            Number.objects.create()
+
+    def test_restrictions_with_no_joining_columns(self):
+        Bracket.objects.create(low_pass_cutoff=4, high_pass_cutoff=7)
+
+        self.assertListEqual(
+            list(Number.objects.filter(low_bracket=1)
+                 .values_list('pk', flat=True)),
+            [1, 2, 3]
+        )
+
+        self.assertListEqual(
+            [b.low_pass_cutoff for b in Bracket.objects.filter(low_pass__in=[3])],
```

Comment:
In `LowPassRelation`, the `_cutoff` field is the concrete field on the model, like the `_id` field for a `ForeignKey`. So if `low_pass_cutoff` is 4, the `low_pass` relation yields related instances with id less than 4.